### PR TITLE
Render Dynamic Featured Articles on Home Page

### PR DESCRIPTION
**Pull request by jestapinski**
This PR pulls out the hardcoded articles on the home page and adds the ability to render featured articles dynamically, similar to with the learn page.

### CODE REVIEW

File: src/utils/get-featured-card-fields.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            title: null,
+            description: null,
+            tags: null,
+        };
+    }
+    const query_fields = article.query_fields;
+    const image = query_fields['atf-image'];
+    const slug = query_fields.slug;
+    const title = dlv(query_fields, ['title', 0, 'value']);
+    const description = dlv(query_fields, [
```
Comment (greggb) [inline src/utils/get-featured-card-fields.js]: Can this use get-nested-value?

File: src/utils/get-featured-card-fields.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        'children',
+        0,
+        'value',
+    ]);
+    const tags = {
+        products: query_fields.products,
+        tags: query_fields.tags,
+        languages: query_fields.languages,
+    };
+    return { image, slug, title, description, tags };
```
Comment (greggb) [inline src/utils/get-featured-card-fields.js]: What are your thoughts  on skipping creating `const`s and just assigning the values here in the return object?

File: src/utils/get-featured-card-fields.js (hunk 1 above)
Comment (jestapinski) [reply src/utils/get-featured-card-fields.js]: Is there a specific rule for using `getNestedValue` vs `dlv`? If it is a node should we default to `getNestedValue`?

Comment (greggb) [reply src/utils/get-featured-card-fields.js]: Yea, that's what I'm thinking. There is a chance for there to be intermediate nodes we don't expect if there is a formatting error in the RST or a writer gets fancy.

**Comment by jestapinski** [comment]
Thanks!

**Pull request merged**